sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use crate::io::emu::{EmuState, EmuValue, Emulator};
+use crate::transport::ti50emulator::Ti50Emulator;
+use crate::transport::{Result, TransportError};
```

Comment:
`transport::Result` no longer exists (since #12144).  In general, all results should be `anyhow::Result`.

Reply:
I have replaced occurrence of Result in other files as well.